src/main/java/com/customerApi/api/controllers/CustomerController.java

```diff
@@ -0,0 +1,86 @@
+package com.customerApi.api.controllers;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.services.CustomerService;
+
+@RestController
+public class CustomerController {
+	private static final Logger logger = LoggerFactory.getLogger(CustomerController.class);
+
+	@Autowired
+	private CustomerService service;
+
+	@PostMapping("/save")
```

Comment:
Naming convention can be better, i am not sure if you read the article I gave you. Please read it carefully this time and fix it
https://restfulapi.net/resource-naming/

Reply:
@PostMapping("/customer-management/custmer")
@PutMapping("/customer-management/custmer")
@GetMapping("/customer-management/custmers")
@GetMapping("/customer-management/custmers/{id}")
@DeleteMapping("/customer-management/custmers")
@DeleteMapping("/customer-management/custmer/{id}")